cli/command/stack/remove_test.go

```diff
@@ -46,7 +46,7 @@ func TestRemoveStack(t *testing.T) {
 		secrets:  allSecrets,
 		configs:  allConfigs,
 	}
-	cmd := newRemoveCommand(test.NewFakeCli(cli, &bytes.Buffer{}))
+	cmd := newRemoveCommand(test.NewFakeCliWithOutput(cli, &bytes.Buffer{}))
```

Comment:
Could just be `NewFakeCli`, but I know you're not trying to exhaustively convert these.